src/App.jsx

```diff
@@ -1,34 +1,41 @@
-import { Routes, Route, Outlet } from "react-router-dom";
-import Home from './pages/Home';
-import About from './pages/About';
-import Contact from './pages/Contact';
-import Team from './pages/Team';
-import Profile from './pages/Profile';
-import NoMatch from './pages/NoMatch';
-import Footer from './components/Footer';
+// import { Routes, Route, Outlet } from "react-router-dom";
```

Comment:
@rch20 do these changes on App.jsx & index.jsx intended or only for testing purposes ?